RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -433,6 +444,10 @@ SiPixelCluster PixelThresholdClusterizer::make_cluster(const SiPixelCluster::Pix
           SiPixelCluster::PixelPos newpix(r, c);
           if (!acluster.add(newpix, theBuffer(r, c)))
             goto endClus;
```

Comment:
This is the original code of the clusterizer. Since its validation would significantly increase the complexity of this PR, I suggest to leave the task of this modification to the person responsible for the maintenance of the clusterizer, in case you feel this is important.

Reply:
sure, a later PR should be fine in this case.  #34705